incubator/zookeeper/templates/statefulset.yaml

```diff
@@ -16,8 +16,11 @@ spec:
         app: {{ include "zookeeper.name" . | quote }}
         release: {{ .Release.Name | quote }}
     spec:
-      {{- if eq .Values.antiAffinity "hard" }}
       affinity:
```

Comment:
I don't like it to have `podAntiAffinity` fix in there plus additional affinities on top. I'd rather remove `podAntiAffinity` and provide full flexibility like so:

```yaml
{{- if .Values.affinity }}
      affinity:
{{ toYaml .Values.affinity | indent 8 }}
{{- end }}
```

There are other charts that do it like this.

Reply:
I like that, I was trying to follow some of the others one. I'll switch it to this if its more of a standard